app-multimedia/mkvtoolnix/autobuild/prepare

```diff
@@ -0,0 +1,4 @@
+abinfo "Setting autotools configure ..."
+AUTOTOOLS_AFTER+=(
```

Comment:
This shouldn't be here, and if this is indeed an Autotools source, then we should not be using a build script in the first place.